server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -4,7 +4,7 @@
 version: "3.8"
 services:
   cms:
-    image: taccwma/core-cms:latest
+    image: taccwma/core-cms:ff737a3
```

Comment:
👍 I like this being pinned. Why this commit?

_P.S. To other reviewers/referencers, `ff737a3` is a commit from Sep 16, 2022 at Core-CMS v3.9.1 ([internal test status](https://confluence.tacc.utexas.edu/x/hop9E))._

Reply:
It's the commit hash from the latest release: https://github.com/TACC/Core-CMS/releases/tag/v3.9.1

Reply:
K. I'm happy with it, cuz it can test that version's compatibility with Portal. I just want us to remember (and now we do) [that version has **not** been tested](https://confluence.tacc.utexas.edu/x/hop9E), but this is a dev compose file, so all good.